### PR TITLE
Simple fix to handle case where there is no es server specified

### PR DESCRIPTION
**Pull request by smalleni**
Signed-off-by: Sai Sindhur Malleni <smalleni@redhat.com>

### CODE REVIEW

**Review by jtaleric — APPROVED** [review]
LGTM, thanks Sai.

File: snafu/run_snafu.py [hunk 1]
```diff
@@ -68,7 +68,7 @@ def main():
     es_settings = {}
     es_settings["server"] = os.getenv("es")
     es_settings["verify_cert"] = os.getenv("es_verify_cert", "true")
-    if ":443" in es_settings["server"] :
+    if es_settings["server"] and ":443" in es_settings["server"]:
```
Comment (rsevilla87) [inline snafu/run_snafu.py:71]: Why this change?? the ES logic only gets executed if "server" env var is specified:
https://github.com/cloud-bulldozer/benchmark-wrapper/blob/ce249862486c78f806a93995a4bc4a612169861c/snafu/run_snafu.py#L73-L80

Comment (jtaleric) [inline snafu/run_snafu.py:71]: hm, should this be `if "server" in es_settings` ?

Comment (rsevilla87) [reply snafu/run_snafu.py:71]: os.getenv("es") returns None if "es" is not set

Comment (smalleni) [reply snafu/run_snafu.py:71]: Yes but you will see `TypeError: argument of type 'NoneType' is not iterable` when trying to check if ":443" in...
